test/integration/affinities.bats

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	swarm_manage_cleanup
+	stop_docker
+}
+
```

Comment:
Would you mind adding an image affinity test since you already made most of the work?